Client/InfluxDBApiException.cs

```diff
@@ -14,4 +16,15 @@ internal InfluxDBApiException(string message, HttpResponseMessage httpResponseMe
     }
 
     public HttpResponseMessage? HttpResponseMessage { get; private set; }
+
+    public HttpResponseHeaders GetHeaders()
+    {
+        return HttpResponseMessage?.Headers;
+    }
+
+    public HttpStatusCode GetStatusCode()
+    {
+        return HttpResponseMessage?.StatusCode ?? 0;
+    }
```

Comment:
This could be written using property getters (invocations in tests  need to be modified accordingly). In any case, `Headers`/`GetHeaders` return value should be nullable (`T?`)

```suggestion

    public HttpResponseHeaders? Headers
    {
        get
        {
            return HttpResponseMessage?.Headers;
        }
    }

    public HttpStatusCode StatusCode
    {
        get
        {
            return HttpResponseMessage?.StatusCode ?? 0;
        }
    }
```

Reply:
Thanks.  A Sensible suggestion.  Updated accordingly.